src/reducer/authReducer.js

```diff
@@ -0,0 +1,39 @@
+export const authInitialState = {
+  ...(localStorage.getItem("encodedToken") !== null
```

Comment:
What does the initial 3 dots do? Is it for spreading purpose?Syntax seems new to me so clarifying it

Reply:
yes.. it's for spreading the object, i'm returning from the ternary operator.